tests/e2e_operations.py

```diff
@@ -383,10 +383,19 @@ def run_config_timeout_check(args):
     LOG.info("No config at all")
     assert not os.path.exists(os.path.join(start_node_path, "0.config.json"))
     LOG.info(f"Attempt to start node without a config under {start_node_path}")
+    config_timeout = 10
     proc = subprocess.Popen(
-        ["./cchost", "--config", "0.config.json", "--config-timeout", "10s"],
+        [
+            "./cchost",
+            "--config",
+            "0.config.json",
+            "--config-timeout",
+            f"{config_timeout}s",
+        ],
         cwd=start_node_path,
         env={"ASAN_OPTIONS": "alloc_dealloc_mismatch=0"},
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
```

Comment:
D'oh, you're right. I've update these to at least write to `out` and `err` files like the full-fat 'remote' does, so that we can check the node output after-the-fact.